src/components/AddedCourses/AddedCoursePane.js

```diff
@@ -101,12 +104,18 @@ class AddedCoursePane extends PureComponent {
         this.setState({ customEvents: AppStore.getCustomEvents() });
     };
 
+    loadScheduleNames = () => {
+        this.setState({ scheduleNames: AppStore.getScheduleNames() });
+    };
+
     getGrid = () => {
         return (
             <>
                 <div className={this.props.classes.titleRow}>
                     <Typography variant="h6">
-                        {`Schedule ${AppStore.getCurrentScheduleIndex() + 1} (${this.state.totalUnits} Units)`}
+                        {`${this.state.scheduleNames[AppStore.getCurrentScheduleIndex()]} (${
+                            this.state.totalUnits
+                        } Units)`}
```

Comment:
Once the values are pulled out to vars, we can simplify this.
```suggestion
                        {`${scheduleName} (${scheduleUnits} Units)`}
```